rules/windows/create_remote_thread/create_remote_thread_win_susp_uncommon_target_image.yml

```diff
@@ -41,6 +41,10 @@ detection:
         TargetImage|endswith:
             - ':\Windows\System32\notepad.exe'
             - ':\Windows\System32\spoolsv.exe'
+    filter_optional_xerox_pjems:
+        SourceImage|endswith: ':\Program Files\Xerox\XeroxPrintExperience\CommonFiles\XeroxPrintJobEventManagerService.exe'
+        StartFunction: 'LoadLibraryW'
+        TargetImage|endswith: ':\Windows\System32\spoolsv.exe'
```

Comment:
```suggestion
        SourceImage: 'C:\Program Files\Xerox\XeroxPrintExperience\CommonFiles\XeroxPrintJobEventManagerService.exe'
        StartFunction: 'LoadLibraryW'
        TargetImage: 'C:\Windows\System32\spoolsv.exe'
```

Reply:
Just FYI (and thank you for following the convention)

The reason behind this change. Is that after internal discussion recently we've opted to revert the change in filters to make broad. Because this can be used as a blind post on multi partition systems.

So we are broad in selection and restrictive in filters.

Reply:
Makes complete sense to me. Thanks!